components/StyledComponents.js

```diff
@@ -151,6 +151,44 @@ export const StyledButton = styled.button`
   }
 `;
 
+export const RoundButton = styled.button`
+  position: fixed;
+  top: 0;
+  right: 0;
+  display: flex;
+  justify-content: center;
+  width: 3rem;
+  height: 3rem;
+
+  align-items: center;
+  margin: 2rem;
+  padding: 0.8rem;
+  border: 2px solid var(--stroke);
+  border-radius: 500px;
+  background: var(--secondary);
+  box-shadow: 6px 5px 0px 0 var(--secondarymid);
+  transition: all 0.3s ease-in-out;
+  color: white;
+  cursor: pointer;
+  font-size: 1.5rem;
+
+  &:hover {
+    transform: scale(1.05);
+    background: var(--tertiary);
+    box-shadow: 6px 5px 0px 0 var(--tertiarymid);
+    color: white;
+
+    a {
+      color: var(--primary);
+    }
+  }
+
+  a {
+    color: white;
+    font-size: 2rem;
+  }
```

Comment:
When do you need an `<a>` tag inside of a button? I think in the case you need something to click on and being redirected, you should rather use a next/link instead.